modules/opengl/src/buffer/framebufferobject.cpp

```diff
@@ -140,7 +140,7 @@ void FrameBufferObject::registerAttachment(GLenum attachmentID, GLuint texId) {
         attachedStencilId_ = texId;
     } else {
         const auto num = enumToNumber(attachmentID);
-        if (num < 0 || num >= maxColorAttachments()) {
+        if (num >= maxColorAttachments()) {
```

Comment:
Vaguely remember @martinfalk adding this check. There for a reason?

Reply:
num is unsigned, hence it is always true...